lib/inspec/resources/security_policy.rb

```diff
@@ -169,9 +169,11 @@ def extract_value(key, val)
       # special handling for string values with "
       elsif !(m = /^\"(.*)\"$/.match(val)).nil?
         m[1]
+      # When there is Registry Values value is not seprated with comma we are not spliting the value for backward compatibility
```

Comment:
Can you explain this comment in more detail with an example? This comment has a few typos as well.